README.md

```diff
@@ -112,6 +127,20 @@ As you may notice, there are two files that need to be defined in order to confi
 
 - `file:///data/config.yml` on Linux
 - `file:c:\\data\config.yml` on Windows
+ 
+## Production deployment
+
+The example module is provided for testing purposes. Only core module should be deployed on production environment (Knot.x application as single verticle without any dependencies).
+
+### Executing fat jar
+
+To run it execute the following command:
+
+```
+java -jar knotx-core-XXX.jar -Dservice.configuration=<path to your service.yml> -Drepository.configuration=<path to your repository.yml>
+```
+
+This will run the server with production settings. For more informations see configuration section.
```

Comment:
Information is uncountable.

For more _information_ see _the_ configuration[LINK?] section.